google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/CloudDebugSettingsEditor.java

```diff
@@ -70,6 +69,8 @@ protected void resetEditorFrom(@NotNull CloudDebugRunConfiguration runConfigurat
               Optional.ofNullable(Services.getLoginService().getActiveUser())
                   .map(CredentialedUser::getEmail)
                   .orElse("")));
+    } else {
+      settingsPanel.setSelectedCloudProject(null);
```

Comment:
why setting to null and not the active project?

Reply:
This editor does not have access to project selector itself, it can just pass the cloud project. And active cloud project is internal to `project` and only used internally by `ProjectSelector`. If you think we can make it cleaner, please let me know. Idea was to hide as many details as possible in `loadActiveProject()`

Reply:
I like it, this will read better! Thanks.